ADAL/src/request/ADAuthenticationRequest+WebRequest.m

```diff
@@ -85,11 +85,12 @@ - (BOOL)verifyStateFromDictionary: (NSDictionary*) dictionary
         
         if (![NSString adIsStringNilOrBlank:authorizationServer] && ![NSString adIsStringNilOrBlank:resource])
         {
-            AD_LOG_VERBOSE_F(@"State", [_requestParams correlationId], @"The authorization server returned the following state: %@", state);
+            AD_LOG_VERBOSE(_requestParams.correlationId, @"The authorization server returned the following state: %@", state);
```

Comment:
state is base64 encoded parameters, which can have resource and authorization server